### ISSUE TITLE
Blank Screen Launch Mac Sierra

### ISSUE THREAD

**Issue by FRINGEMASTERR** [SFTtech/openage#874]
After finding the packages, renamed form those cited in the MAC OS Yosemite, the assets were converted and it did launch but with a blank run window.:

```
"INFO [T17007461912808498372] [py] launching openage v0.3.0-972-g8313de75 
INFO [T17007461912808498372] [py] compiled by Clang 4.0.1 
INFO [T17007461912808498372] [py] running in DEVMODE 
INFO [T17007461912808498372] launching engine with Path(Union(Directory(/Users/FRINGEMASTER/openage/cfg).root @ (b'cfg',), Directory(/Users/FRINGEMASTER/openage/assets).root @ (b'assets',)):) and fps limit 0 
INFO [T17007461912808498372] SDL audio subsystems initialized 
INFO [T17007461912808498372] Using audio device: default [freq=48000, format=32784, channels=2, samples=4096] 
INFO [T17007461912808498372] Initialized SDL video subsystems. 
INFO [T17007461912808498372] Compiled with Qt 5.9.1 and run with Qt 5.9.1
INFO [T17007461912808498372] Falling back to separate render context for GUI 
INFO [T17007461912808498372] loading configuration files... 
INFO [T17007461912808498372] [py] loading config file [Union(Directory(/Users/FRINGEMASTER/openage/cfg).root @ (b'cfg',), Directory(/Users/FRINGEMASTER/openage/assets).root @ (b'assets',))]:cfg/keybinds.oac... 
INFO [T17007461912808498372] Loading time [engine]: 0.747 s 
INFO [T17007461912808498372] Loading time [game]: 0.849 s INFO [T3356274813759464202] Loading game specification files... 
INFO [T3356274813759464202] Loaded multi-csv file: 45484 sub-files 
INFO [T3356274813759464202] Loading textures... 
INFO [T3356274813759464202] Loading sounds... 
INFO [T3356274813759464202] Loading time [data]: 22.272 s Z [5]+ Stopped ./run Jaquess-MacBook-Air:openage FRINGEMASTER$ sudo ./run Password: 
INFO [T17007461912808498372] [py] launching openage v0.3.0-972-g8313de75 
INFO [T17007461912808498372] [py] compiled by Clang 4.0.1 INFO [T17007461912808498372] [py] running in DEVMODE 
INFO [T17007461912808498372] launching engine with Path(Union(Directory(/Users/FRINGEMASTER/openage/cfg).root @ (b'cfg',), Directory(/Users/FRINGEMASTER/openage/assets).root @ (b'assets',)):) and fps limit 0 
INFO [T17007461912808498372] SDL audio subsystems initialized 
INFO [T17007461912808498372] Using audio device: default [freq=48000, format=32784, channels=2, samples=4096] 
INFO [T17007461912808498372] Initialized SDL video subsystems. 
INFO [T17007461912808498372] Compiled with Qt 5.9.1 and run with Qt 5.9.1 
INFO [T17007461912808498372] Falling back to separate render context for GUI 
INFO [T17007461912808498372] loading configuration files... 
INFO [T17007461912808498372] [py] loading config file [Union(Directory(/Users/FRINGEMASTER/openage/cfg).root @ (b'cfg',), Directory(/Users/FRINGEMASTER/openage/assets).root @ (b'assets',))]:cfg/keybinds.oac... 
INFO [T17007461912808498372] Loading time [engine]: 0.725 s 
INFO [T17007461912808498372] Loading time [game]: 0.825 s INFO [T14196087192439472191] Loading game specification files... 
INFO [T14196087192439472191] Loaded multi-csv file: 45484 sub-files 
INFO [T14196087192439472191] Loading textures... 
INFO [T14196087192439472191] Loading sounds... 
INFO [T14196087192439472191] Loading time [data]: 19.602 s Z[5] Killed: 9 ./run
[6]+ Stopped sudo ./run
```

Make test returned one failed test:
 
```
py util.fslike.test.test Test failed
Traceback (most recent call last):
File "/Users/FRINGEMASTER/openage/openage/testing/main.py", line 129, in main testfun()
File "/Users/FRINGEMASTER/openage/openage/util/fslike/test.py", line 185, in test test_case_ignoring(root_path, root_dir)
File "/Users/FRINGEMASTER/openage/openage/util/fslike/test.py", line 165, in test_case_ignoring assert_value(root_path["A"].is_file(), False)
File "/Users/FRINGEMASTER/openage/openage/testing/testing.py", line 36, in assert_value raise TestError("unexpected result: " + repr(value))
openage.testing.testing.TestError: unexpected result: True
```

WHen I recompile openage, I find this as an error.  I thinkI remeber it testing successful last time, but I do not remember.:

```
-- The CXX compiler identification is Clang 4.0.1
-- Check for working CXX compiler: /usr/local/opt/llvm/bin/clang++
-- Check for working CXX compiler: /usr/local/opt/llvm/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAVE_THREAD_LOCAL_STORAGE
-- Performing Test HAVE_THREAD_LOCAL_STORAGE - Failed
-- Using python interpreter: /usr/local/opt/python3/bin/python3.6
-- Found Python: /usr/local/opt/python3/bin/python3.6 (Required is at least version "3.4")
-- Found Cython: /usr/local/opt/python3/bin/python3.6 -m cython (found suitable version "0.25.2", minimum required is "0.25")
-- Found numpy: /usr/local/lib/python3.6/site-packages/numpy/core/include
-- Checking python3 module PIL.Image - Success
-- Checking python3 module PIL.ImageDraw - Success
-- Checking python3 module numpy - Success
-- Checking python3 module pygments - Success
-- Checking utility program opusenc - Success
-- Found Doxygen: /usr/local/bin/doxygen (found version "1.8.13") found components:  doxygen missing components:  dot
CMake Warning at buildsystem/doxygen.cmake:17 (message):
  graphviz dot couldn't be found, you won't have cool graphs in the docs.
Call Stack (most recent call first):
  buildsystem/doxygen.cmake:54 (doxygen_init)
  CMakeLists.txt:96 (include)
```

**Comment by ChipmunkV** [comment]
Probably need to do #852 first.

**Comment by TheJJ** [comment]
It's very strange that the `HAVE_THREAD_LOCAL_STORAGE` test fails for you. Also, that the `util.fslike.test.test` test fails, the `A` file should not exist. On Linux machines, this test passes, I just checked on my machine, `A` does not exist there indeed.

And yes, #852 is most likely the remaining missing piece for mac systems.

**Comment by TheJJ** [comment]
I opened #876 for the test problem. Your black screen should be addressed in #852.
Why no thread local storage can be used, I can't tell.
